### PR TITLE
#5477 - Create Vote Weight Module

### PR DESCRIPTION
**Pull request by Miaplacidus**
This PR completes ticket #5477, which asks for the creation of a vote weight module to be placed in the side bar navigation. The relevant designs for both desktop and mobile can be found [here](https://www.figma.com/file/KOEVNGXStBhmX3NnYq7MxP/Community-Staking?type=design&node-id=1153-224753&mode=design&t=BVH8vX8wmVfLSNNr-4).

## Link to Issue
Closes: #5477 

## Description of Changes
- creates a module listing the stake a user has in a community and the value of that stake in the relevant denomination
- the module includes an informative popover that appears upon hovering over the 'info' icon on desktop, and upon tap on mobile devices
- the aforementioned popover closes when upon clicking the 'x' in the upper right hand corner on mobile

## Test Plan
- navigate to the `/components` page and scroll to the Vote Weight Module section
- check that expected information is within module
- hover over info button to see info popover
- use mobile view on your browser to tap on info button. Also check that info popover has close button and that it closes the popover

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWVoteWeightModule/CWVoteWeightModule.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  stakeNumber: number;
+  stakeValue: number;
+  denomination: string;
+};
+
+export const CWVoteWeightModule = ({
+  voteWeight,
+  stakeNumber,
+  stakeValue,
+  denomination,
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWVoteWeightModule/CWVoteWeightModule.tsx:21]: I am not sure about "currency" here but if this always be "ETH", we can make "ETH" as default here

File: packages/commonwealth/client/styles/mixins/text.scss [hunk 2]
```diff
@@ -26,7 +26,7 @@
   font-family: $font-family-neue-haas-unica;
   font-size: 12px;
   letter-spacing: 0.02em;
-  line-height: 18px;
```
Comment (masvelio) [inline packages/commonwealth/client/styles/mixins/text.scss:29]: this is a global change for whole project for text type `caption`. Do we want this?

**Review by masvelio — APPROVED** [review]
lgtm

**Review by Israellund — APPROVED** [review]
lgtm

File: packages/commonwealth/client/styles/mixins/text.scss (hunk 2 above)
Comment (Miaplacidus) [reply packages/commonwealth/client/styles/mixins/text.scss:29]: I changed this based on updates to the typography figma.

**Comment by Miaplacidus** [comment]
@masvelio I've re-set the line-height on captions to 18px. There's an inconsistency in the typography Figma that I'm resolving with @jessmart1213, but I think 18 px is the actual line-height, i.e. 16px was set in error.

**Pull request merged**